packages/sdk/src/utils/message-utils.ts

```diff
@@ -46,11 +55,30 @@ export const migratedWithdrawalGasLimit = (
   chainID: number
 ): BigNumber => {
   // Compute the gas limit and cap at 25 million
```

Comment:
This would be slightly safer with `ethers.BigNumber` instead of builtin `number`